### PR TITLE
Added computer vision tasks

### PR DESCRIPTION
**Pull request by merveenoyan**
Previous PR was in my fork so thought it'd be easier if I do it from a branch. Added computer vision task datasets according to HF tasks.

### CODE REVIEW

**Review by lhoestq — APPROVED** [review]
Thanks !

For future references, note that the top level names come from here:
https://github.com/huggingface/huggingface_hub/blob/main/js/src/lib/interfaces/Types.ts

let us know what you think @mariosasko

File: src/datasets/utils/resources/tasks.json [hunk 1]
```diff
@@ -98,6 +98,43 @@
...
     },
+    "object-detection": {
+        "description": "tasks related to detecting instances of objects from a particular class in an image",
+        "options": [
+            "face-detection",
+            "other"
+        ]
+    },
+    "image-to-text": {
+        "description": "tasks related to generating images from an existing dataset",
```
Comment (osanseviero) [inline src/datasets/utils/resources/tasks.json:109]: maybe this is clearer?
```suggestion
        "description": "tasks related to generating images from a given text",
```

Comment (osanseviero) [reply src/datasets/utils/resources/tasks.json:109]: Actually, this is the other way around I think. This should be generating text from a given image

**Review by mariosasko — APPROVED** [review]
I agree with @osanseviero's comment, but other than that LGTM! Well done!

**Review by osanseviero — APPROVED** [review]
Very cool!

**Pull request merged**